lighthouse-cli/chrome-launcher.ts

```diff
@@ -136,6 +136,11 @@ class ChromeLauncher {
 
   spawn(execPath: string): Promise<any[]> {
     return new Promise((resolve, reject) => {
+      if (this.chrome) {
+        log.verbose('ChromeLauncher', `Chrome already running with pid ${this.chrome.pid}.`);
```

Comment:
(the tradeoff being that it adds warning noise to a use case like your own. A compromise might be just `log.log`)